### DISCUSSION TITLE
Which Admin Dashboard it is ?

### DISCUSSION THREAD

**Discussion by askaralimkndr** [uasoft-indonesia/badaso#497] [General]
Hi,
Product is really amazing, i am very curies to know which admin template used ? is it bootstrap 5 ? or Tailwind CSS ?
Thank you

**Comment by rizkiheryandi** [comment]
badaso v1.x & v2.x using [vuesax](https://lusaxweb.github.io/vuesax/)

v3.x will using vuetail
